runtime/Makefile

```diff
@@ -0,0 +1,152 @@
+##
```

Comment:
Do you actually need this? Doesn't "make run" from the fmc directory already build the app firmware?

Reply:
It does, but the FMC doesn't boot into RT right now. So I just wanted to replicate that Makefile here and boot directly into RT.

Maybe it would make more sense to

* Move fmc/Makefile to the root
* Add a target to boot into runtime directly

Reply:
What am I missing here? Isn't this the runtime firmware?

```
fmc$ make run
<snip>
 / ___|__ _| (_)_ __ | |_ _ __ __ _  |  _ \_   _|
| |   / _` | | | '_ \| __| '__/ _` | | |_) || |
| |__| (_| | | | |_) | |_| | | (_| | |  _ < | |
 \____\__,_|_|_| .__/ \__|_|  \__,_| |_| \_\|_|
               |_|

[rt] FHT Marker: 0x54484643
<snip>
fmc$
```

Reply:
Also, "make run" isn't very useful, since you can't manipulate the mailbox there. Most testing should be done against the HwModel from Rust.

Reply:
Oh you're right. For whatever reason it wasn't working for me yesterday. Ok, will ditch this and just use the fmc makefile